### ISSUE TITLE
Xendit class implementation to allow multiple class instances

### ISSUE THREAD

**Issue by kevindavee** [xendit/xendit-java#30]
Right now, the Xendit class is using a static property for storing the API key. It becomes hard when we want to have multiple API keys in usage. We need to back and forth reassign the API key value.

Having the Xendit class to have a client builder way of implementation will enable developers to use the Xendit class, instantiate to multiple variables, and use different API keys.

Example:
```java
XenditClient client = new XenditClient.Builder()
            .apiKey("YOUR_API_KEY")
            .build();
```

**Comment by mistahenry** [comment]
Is this being actively worked on? I'm using this library and just started to expand markets from ID to the PH. Xendit required that I have two separate accounts, one per country. The assumption that the user of this library only has one API key means it's not really usable at this time for me.

**Comment by edwinyoyada** [comment]
Hi @mistahenry, we also understand your concern regarding this issue. From Xendit's perspective, the nearest timeline is end of September. If you want to open PR, we're gladly to review and push this forward.

**Comment by edwinyoyada** [comment]
resolved in v1.16.0
